addons/mass_mailing_themes/views/mass_mailing_themes_templates.xml

```diff
@@ -6,13 +6,16 @@
             <div data-name="newsletter" title="Event Promo" data-img="/mass_mailing_themes/static/src/img/theme_imgs/newsletter_thumb" data-images-info='{"logo": {"format": "png"}, "all": {"module": "mass_mailing_themes"}}'>
                 <t t-call="mass_mailing_themes.theme_newsletter_template"/>
             </div>
+            <div data-name="textnews" title="Newsletter" data-layout-styles="background-color: rgb(255, 255, 255) !important;" data-img="/mass_mailing_themes/static/src/img/theme_imgs/malice_thumb" data-images-info='{"logo": {"format": "png"}, "all": {"module": "mass_mailing_themes"}}'>
```

Comment:
I'm a little bit confused here. The other themes seem to follow some naming convention such that "newsletter" for example has "newsletter_thumb". But this one has "malice_thumb" instead of "textnews_thumb". Is this intended ? Are you malicious ? 😈